config/locales/devise.en.yml

```diff
@@ -219,21 +219,21 @@ en:
         message: To create an account, you need to <a class="govuk-link" href="%{link}">answer a few questions in the Brexit checker</a> and subscribe to email updates.
       transition_emails:
         description: |
-          <p class="govuk-body">The emails will give you a personalised list of Brexit actions for you, your family and your business.</p>
+          <p class="govuk-body">The emails will give you a personalised list of actions for you, your family and your business.</p>
           <p class="govuk-body">You will only get updates if:</p>
           <ul class="govuk-list govuk-list--bullet">
-            <li>there are changes to your results</li>
+            <li>there are changes to your Brexit checker results</li>
             <li>questions or results are added which may affect you</li>
           </ul>
           <p class="govuk-body">You’ll never get more than one email a day. The email will include all the updates made that day.</p>
         fields:
           emailsignup:
-            heading: Do you want to receive emails about the Brexit checker?
+            heading: Do you want to receive emails about the Brexit?
```

Comment:
This, and the one below, should be "about Brexit?"

Reply:
woops, quite right, let me change that before Ale sees!